src/data/RoboticsPhotoGalleryInfo.tsx

```diff
@@ -1,30 +1,31 @@
 import { RMC_PHOTO_GALLERY } from '../tools/Constants'
 import { GalleryImage } from '../pages/general/photo-gallery/PhotoGalleryStyles'
-
```

Comment:
I wouldn't add this as a new variable. I would just put it inline like the others like on like 8 and 9